src/components/Page.vue

```diff
@@ -1,6 +1,7 @@
 <template>
-  <div
-    :class="[fill, { desktop: !$globals.IS_MOBILE_DEVICE }]"
+  <component
```

Comment:
```suggestion
  <Component
```
this is not a native HTML element, so I think it should be capitalized